apps/desktop-wallet/src/storage/addresses/addressesSelectors.ts

```diff
@@ -30,9 +30,11 @@ export const makeSelectAddresses = () =>
 
 export const selectDefaultAddress = createSelector(
   selectAllAddresses,
-  (addresses) => addresses.find((address) => address.isDefault) || addresses[0]
+  (addresses) => addresses.find((address) => address.isDefault) || addresses.at(0)
```

Comment:
Side note about this (don't remember if we talked about that already), but there's a lint rule to output a warning when we do things like addresses[0]. I tried it, but it's pretty strict / restrictive / dumb. Never thought about using `addresses.at(0)` because I didn't know it'd return either the type or `undefined`, nice finding!

Reply:
Yeah, I think the rule is the one that we currently have an open PR about and indeed it's very restrictive and we'd need to make a ton of changes.